packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -14,7 +14,7 @@ class MongodbCorePlugin extends DatabasePlugin {
     super.configure(config)
     this.config.heartbeatEnabled = coalesce(
       config.heartbeatEnabled,
-      process.env.DD_TRACE_MONGODB_HEARTBEAT_ENABLED,
+      process.env.DD_TRACE_MONGODB_HEARTBEAT_ENABLED === 'false' ? false : null,
       true
```

Comment:
you'll also need to require at the top of the file: ``const { isTrue } = require('../../dd-trace/src/util)
```suggestion
      isTrue(process.env.DD_TRACE_MONGODB_HEARTBEAT_ENABLED)
```

Reply:
https://github.com/DataDog/dd-trace-js/pull/5558/commits/ee518c73f1e442163585f3983b25611e615b6a2a